src/app.tsx

```diff
@@ -24,7 +24,7 @@ window.addEventListener('load', () => {
   const api = apiFactory(serverApi, new LocalApi());
   const history = createHashHistory();
   const router = new RouterStore(history);
-  // @ts-ignore - Need to provide proper typings for actions
+  // @ts-expect-error - Need to provide proper typings for actions
```

Comment:
`@ts-expect-error` is better than `@ts-ignore`, see more: https://typescript-eslint.io/rules/prefer-ts-expect-error/